.mergify.yml

```diff
@@ -0,0 +1,72 @@
+pull_request_rules:
+  - name: Automatic merge for main branch
+    conditions:
+      - base=main
+      - "check-success=Shellcheck"
+      - "check-success=Lint"
+      - "check-success=Tidy"
+      - "check-success=Mocks"
+      - "check-success=Chart checks"
+      - "check-success=Go build"
+      - "check-success=Go test"
+      - "check-success=Test various Envoy + SMI configuration scenarios"
+      - "check-success=Integration Test with Tresor, SMI traffic policies, and egress disabled"
+      - "check-success=Go test e2e (1)"
+      - "check-success=Go test e2e (2)"
+      - "check-success=Go test e2e (3)"
+      - "check-success=Go test e2e (4)"
+      - "check-success=DCO"
+      - "check-success=netlify/osm-docs/deploy-preview"
+      - "#approved-reviews-by>=2"
+      - label=automerge
+      - -label=wip
+      - -label=do-not-merge/hold
```

Comment:
How come the `do-not-merge/hold` and `label=wip` have an extra `-` in front of them but `label=automerge` doesn't?

Reply:
Negate label syntax?

Reply:
Yep it's like Shashank mentioned, @michelleN `-label=wip` is only true when the PR doesn't have the `wip` label whereas `label=automerge` is only true when the PR does have the `automerge` label

Reply:
Didn't know that. Thank you!